python-checks/src/test/resources/checks/identityComparisonWithCachedTypes.py

```diff
@@ -152,3 +152,7 @@ def comparison_to_none():
     "str" is None
     123 is None
     None is "str"
+
+def potential_null_symbols():
+    type(arr) is tuple # see SONARPY-2259
+    some_thing is other_thing # see SONARPY-2260
```

Comment:
I'm not sure it makes sense to reference these other tickets here.
I think there is a distinction to be made between:
- A test case added to highlight an unwanted behavior of the rule (e.g FP/FN).
- A test case added to ensure proper test coverage, because of an unrelated limitation.

In the first case, it is normal to reference the ticket that would fix the unwanted behavior. In the second, fixing the ticket will not change the test result anyway, and the reference to the ticket is likely to stay in the code because of that (c.f Michael's comments on our weekly about stale ticket references in our code). Simply having the case for coverage is enough.

That being said, I think it would make sense to have actual frontend tests for our symbol tables (most likely in `SymbolTableBuilderV2Test`) that highlight these behaviors and link to a ticket.